test/basis.test.js

```diff
@@ -142,7 +142,7 @@ describe("Basis Test", () => {
 
         test("MediaQuery 1024 selectors should be included", () => {
             const missingSelectors = [];
-            const selectorPrefix = "all and (min-width: 1024px)===";
```

Comment:
Not sure about that. I want to be sure that those terms won't excluded in the extraction process.

Reply:
In nocss case, `all and` will be dropped by puppeteer.